_emails/montgomery_county.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Montgomery County
+permalink: "/montgomery_county"
+name: Letter to the County Executive and Council
+state: MD
+city: Montgomery County
+layout: email
+recipients:
+- marc.elrich@montgomerycountymd.gov
+- councilmember.Katz@montgomerycountymd.gov
+- councilmember.Hucker@montgomerycountymd.gov
+- councilmember.albornoz@montgomerycountymd.gov
+- councilmember.Friedson@montgomerycountymd.gov
+- councilmember.Glass@montgomerycountymd.gov
+- councilmember.jawando@montgomerycountymd.gov
+- councilmember.Navarro@montgomerycountymd.gov
+- councilmember.Rice@montgomerycountymd.gov
+- councilmember.Riemer@montgomerycountymd.gov
+subject: MD Resident for Defunding the Montgomery County Police Department
+body: |-
+Dear Montgomery County Officials,
+
+My name is [YOUR NAME], and I am a resident of Montgomery County, MD. This past week, our nation has been gripped by protests calling for rapid and meaningful change with regard to police behavior, an end to racism and anti-blackness, and immediate reform in how black people are treated in America. Our county has been at the forefront of much of this action.
+
+The brutal, senseless, and violent murder of George Floyd at the hands of police officers derelict in their duties to protect and serve is deeply disturbing. We, as a nation, are in need of a drastic overhaul in policing. Systems of oppression must be undone, and reallocating much of the budget for the MPD to programs and city-led initiatives that support education, rehabilitation, public health, and community-oriented initiatives is a goal that must be achieved.
+
+Montgomery County's Police is not exempt from racial issues. There is story after story of black people being harassed, searched and arrested for simple being black.
+
+How many of you have been arrested for helping a drunk neighbor home?
+What about standing outside of a McDonalds?
+What about walking down the street?
+Will Jawando himself was stopped for driving while black. If even elected officials are not safe from this treatment, how can the average person have any sense of safety and trust in the police?
+
+This is not acceptable behavior for people expected to uphold the law and keep all members of the community safe. There are consequences to actions. Even if you wear a badge.
+
+Sincerely,
+[YOUR NAME]
+[YOUR ADDRESS]
+[YOUR EMAIL]
+[YOUR PHONE NUMBER]
```

Comment:
body text needs to be updated. this is consistent against a lot of the PRs you have made - please backfill them all with this fix if possible!

Reply:
On it now and fixing all of these now! Thanks for calling it out.